tests/unit/dialog/tests.html

```diff
@@ -430,6 +447,214 @@
+
+            simulateEvent(header, "dblclick", { which: 1, pageX: dim.right, pageY: vCenter });    
+            equal(dialog.options.state, "opened","The dialog is opened"); 
+
+            $("#dialog5").igDialog("option", "enableDblclick", true);
+            simulateEvent(header, "dblclick", { which: 1, pageX: dim.right, pageY: vCenter });    
+            equal(dialog.options.state, "maximized","The dialog is maximized"); 
+
+            $("#dialog5").igDialog("option", "state", "minimized");
+            simulateEvent(header, "dblclick", { which: 1, pageX: dim.right, pageY: vCenter });    
+            equal(dialog.options.state, "opened","The dialog is not opened after being minimized");  
+
+            simulateEvent(header, "touchstart", { which: 1, pageX: dim.right, pageY: vCenter, touches: 1});    
+            simulateEvent(header, "touchmove", { which: 1, pageX: dim.right + 10, pageY: vCenter });    
+            simulateEvent(header, "touchend", { which: 1, pageX: dim.right + 10, pageY: vCenter });    
+
+            $("#dialog5").igDialog("option", "showHeader", false);           
+                  
+        }); 
+
+        test("Resize horizontally", function () {            
+            $("#dialog5").igDialog({
+                draggable: true,
+                resizable: true,
+            });
+
+            var dialog = $("#dialog5").data().igDialog; 
+            $("#dialog5").igDialog("option", "modal", false);
+            dialog.moveToTop();            
+
+            var seResizer = $('#dialog5').find('.ui-resizable-se'),
+                dim = getElementDimensions($('#dialog5')),
+                vCenter = (dim.top + dim.bottom) / 2,
+                xCenter = (dim.left + dim.right) / 2;
+
+            simulateEvent(seResizer[0], "mousedown", { which: 1, pageX: dim.right, pageY: vCenter });
+            simulateEvent(seResizer[0], "mousemove", { which: 1, pageX: dim.right - 100, pageY: vCenter});
+            simulateEvent(seResizer[0], "mouseup", { which: 1, pageX: dim.right - 10, pageY: vCenter });
+
+            var resultDim = getElementDimensions($('#dialog5'));
+            equal(resultDim.right, dim.right,"The dialog is not opened after being minimized");  
+
+            simulateEvent(seResizer[0], "mousedown", { which: 1, pageX: dim.right, pageY: vCenter });
+            simulateEvent(seResizer[0], "mousemove", { which: 1, pageX: dim.right , pageY: vCenter + 1000});
+            simulateEvent(seResizer[0], "mouseup", { which: 1, pageX: dim.right - 10, pageY: vCenter });
+        }); 
+
+        test("InputName", function () {            
+            $("#dialog6").igDialog({     
+                inputName: "input1",
+                position: [100,100]
+            });
+
+            $("#dialog6").igDialog("option", "inputName", "input2");
+            $("#dialog6").igDialog("option", "state", "minimized");
+
+            ok($("#input1")[0].value.indexOf("s1:w300") !== -1,"The saved value is correct");  
```

Comment:
Maybe also check the state contains `p100,100` matching the initial options too :)